webapp/channels/src/components/team_sidebar/team_sidebar.tsx

```diff
@@ -264,6 +264,7 @@ export default class TeamSidebar extends React.PureComponent<Props, State> {
                     }
                     content={plusIcon}
                     switchTeam={this.props.actions.switchTeam}
+                    displayName='Other teams you can join'
```

Comment:
These strings should be translatable since they're currently just available in English. Could you change these to use `intl.formatMessage` similar to line 249 above?

That line uses the deprecated function `Utils.localizeMessage` currently, but if you update your branch to the latest master, it'll use the proper method for translation